containers/daap-push-to-catalogue/src/var/task/push_to_catalogue.py

```diff
@@ -0,0 +1,104 @@
+import json
+import structlog
+import logging
+import os
+
+import boto3
+from data_platform_catalogue import CatalogueClient, CatalogueError
+from data_platform_catalogue import DataProductMetadata as omdProductMetadata
+from data_platform_catalogue import TableMetadata
+
+
+def handler(event, context):
+    # can't use daap-python-base as it's python 3.11 and need 3.10 for
+    # data_platform_catalogue, hence we can't use DataPlatformLogger here
+    structlog.configure(
+    wrapper_class=structlog.make_filtering_bound_logger(logging.INFO),
+    processors=[
+        structlog.processors.EventRenamer(to="message"),
+        structlog.processors.TimeStamper(
+            fmt="%Y-%m-%d %H:%M:%S", key="date_time"
+        ),
+        structlog.processors.add_log_level,
+        structlog.processors.dict_tracebacks,
+        structlog.processors.CallsiteParameterAdder(
+            parameters={structlog.processors.CallsiteParameter.FUNC_NAME},
+            additional_ignores=["data_platform_logging"],
+        ),
+        structlog.processors.JSONRenderer(),
+    ],
+)
+
+    metadata = event["metadata"]
+    version = event.get("version")
+    data_product_name = event["data_product_name"]
+    table_name = event.get("table_name")
+
+    logger = structlog.get_logger(**{"data_product_name":data_product_name, "table_name": table_name})
```

Comment:
```suggestion
    logger = structlog.get_logger(data_product_name=data_product_name, table_name=table_name)
```

this should be equivalent I think